core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -518,6 +540,12 @@ private void appendByteColumn(Record record, int columnIndex) {
         responseAsciiSink.putLenEx(a);
     }
 
+    private void appendCharColumn(Record record, int columnIndex) {
```

Comment:
This method is introduced but not tested. How does this map to PG type?

Reply:
sorry, removed as not used